### ISSUE TITLE
multiple  output  explainer

### ISSUE THREAD

**Issue by asheetal** [ModelOriented/DALEX#92]
I am working on a keras model that  where Y is a 2D vector of 100 columns (similar to MNIST 0-9 digits classification  with 10 columns). How  do I  run explainer/variable-importance if there are multiple output columns in model?  The tutorial  I  see  is for single column output  only

I  tried  feeding the whole Y  matrix  in 
DALEX::explain  followed  by
DALEX::variable_importance

The output was ambiguous.

**Comment by pbiecek** [comment]
Do you have some reproducible example?

We have some examples here: https://pbiecek.github.io/DALEX/reference/feature_response.html
(random forest returns scores for three classes), but not all explainers handle multivariate response, 
it's something that we need to add to the roadmap

**Comment by asheetal** [comment]
It is a huge unweildy code. Hope to publish entire code soon on  Github  at the end of the research paper. For example here is a  code snippet from the Lime package

explanation_keras <- lime::explain (
  x = test_case,
  explainer    = explainer_keras,
  n_labels     = 1, # explaining a `single class`(binary)
  n_features   = 5)

This above code understands that  there is only label  in  this model. In my  code,  the model has  100  labels using categorical crossentropy. (sparse_categorical_crossentropy support would  be  even better). I  have not tried the above command  yet but maybe Lime would theoretically allow  me  to run  one test  case for 100  sparse labels. Will try soon. Was hoping to  find a way  to aggregate  all 100 labels in one DALEX  command. Would  be a a nightmare to build 100 different models, one for each  label.

**Comment by pbiecek** [comment]
Related to #128 and cross entropy loss function.
@maksymiuks Let's move these examples into a new vignette
